src/SourceBuild/tarball/patches/msbuild/0003-Restore-building-all-TFMs-for-source-build-to-suppor.patch

```diff
@@ -0,0 +1,32 @@
+From b7f0c8d910b5fa110be7f34816c4934f61643d08 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Wed, 20 Oct 2021 15:19:29 -0500
+Subject: [PATCH] Restore building all TFMs for source-build to support NuGet
+ to support templating.
+
+---
```

Comment:
Is there an issue letting MSBuild know about this?

Reply:
No, We should push this to msbuild once we get the must have's for GA.

Reply:
Ok, I'll work on that. (I don't think this needs to block the PR, in the interest of getting a build through.)